@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -152,7 +152,7 @@ export default {
       selectedVariationId: null,
       currentGalleyImg: 1,
       isOnCart: false,
-      qntToBuy: 1,
+      qntToBuy: this.product.min_quantity || 1,
```

Comment:
Acho que o melhor seria você setar o `this.qntToBuy` no hook `created`, depois de já certificado que o corpo do produto veio na prop ou foi buscado na API

Reply:
ok